### DISCUSSION TITLE
Mastodon Analytics - api.joinmastodon.org/statistics down?

### DISCUSSION THREAD

**Discussion by erickhun** [mastodon/mastodon#23626] [Show and tell]
Hi, I've made  https://mastodon-analytics.com 

It was using https://api.joinmastodon.org/statistics , but since the 10Feb, the endpoint seems down. 
I liked since it was provided by join.mastodon.com, and felt more official than iterating over all servers that could potentially give "misleading stats".  Any hope to bring this endpoint back @Gargron ?

**Comment by Gargron** [comment]
I need to fix performance of the endpoint. I'll bring it back as soon as I can. See status on https://status.joinmastodon.org
